lbrynet/dht/node.py

```diff
@@ -51,7 +52,7 @@ class Node(object):
 
     def __init__(self, node_id=None, udpPort=4000, dataStore=None,
                  routingTableClass=None, networkProtocol=None,
-                 externalIP=None):
+                 externalIP=None, dataDirectory=None, fileName=None):
         """
         @param dataStore: The data store to use. This must be class inheriting
                           from the C{DataStore} interface (or providing the
```

Comment:
need better docstring for dataDirectory (describe what it does when None, etc..)  , need docstring for fileName